processor/src/test/resources/fixtures/org/mapstruct/ap/test/bugs/_913/DomainDtoWithNvmsDefaultMapperImpl.java

```diff
@@ -39,18 +39,15 @@ public Domain create(Dto source) {
         Domain domain = new Domain();
 
         if ( source != null ) {
-            Set<Long> set = stringListToLongSet( source.getStringsInitialized() );
```

Comment:
Ok.. shall we add everything beneath `Distinguish were?`  to the java doc of the new builder :smile: